### ISSUE TITLE
Insert frame should only affect following audio levels.

### ISSUE THREAD

**Issue by melieconiek** [tahoma2d/tahoma2d#820]
**Is your feature request related to a problem? Please describe**
Working with lots of audio samples in Tahoma can be trying. When audio levels are overlapping, you often only want to shift all of the following audio levels forward without affecting any previous samples. 

It be nice if using the insert/remove frame (and the selection handles) only affected audio levels that are positioned after the cursor. Audio levels can be split or trimmed in other ways. Working around this issue often requires shifting audio into other columns and locking them, which is cumbersome and can easily lead to errors that misalign audio if you're not careful.

![b](https://user-images.githubusercontent.com/42298081/137586568-94cfd8a8-ccd5-43b7-8684-41374eefbd34.gif)

**Describe the solution you'd like**

**Comment by manongjohn** [comment]
Insert Frame is for Scene Frames, not specific Level Frames and is designed to shift ALL cells down in unlocked columns after the current frame.  When doing this, it auto extends the current frame to the shifted frames.  Same for Remove Frame.

Is the issue the audio frame is being auto-extended/copied, causing the equivalent to a stutter,?  Should the audio not auto-extend in this case so gap is created in the audio?

**Comment by melieconiek** [comment]
In nearly all cases you'd want an individual audio sample to remain continuous if you're adjusting exposures underneath. You can lock the audio column, but you'd also want the following audio samples to to shift down. If they don't, you've got a ton of audio that's no longer synced with the drawings below. This requires a lot of finagling that should be unnecessary.

Splitting the audio level instead of copying the last frame would be a better behavior, but you'd still be chopping up your audio bunch when you didn't want to.

Only affecting the following audio levels wouldn't be that bad. If you wanted to split or trim an audio level you still could with a selection and drag, or by deleting a portion of it.

**Comment by manongjohn** [comment]
After a quick discussion to clarify...this request assumes the following:

1. There are multiple separate audio segments in the same column
2. If you Insert/Remove frame in the middle of an existing audio segment...the existing audio segment is not touched, but the other separate segments that follow later will shift accordingly.

**Comment by melieconiek** [comment]
Yes, that's correct. It may be best to make it optional. You may want the current behavior depending on how you're handling audio.

If selection handles are made to affect audio levels, this request would apply to that as well.
